crates/biome_css_analyze/src/lint/nursery/no_unknown_property.rs

```diff
@@ -70,7 +70,10 @@ impl Rule for NoUnknownProperty {
     fn run(ctx: &RuleContext<Self>) -> Option<Self::State> {
         let node = ctx.query();
         let property_name = node.name().ok()?.text().to_lowercase();
-        if !is_known_properties(&property_name) && !vendor_prefixed(&property_name) {
+        if !is_known_properties(&property_name)
+            && !vendor_prefixed(&property_name)
+            && !property_name.starts_with("--")
```

Comment:
Given the performance boost of adding this gate to the lint rule, I suggest changing the order of conditions into the following, because checking `starts_with("--")` is cheaper then multiple binary searching. I benched it locally and there's a very tiny performance gain (3%\~6% for `bootstrap.css`, 0\~1% for others).
```suggestion
        if !property_name.starts_with("--")
            && !is_known_properties(&property_name)
            && !vendor_prefixed(&property_name)
```

Reply:
YES, I appreciate your feedback!!